experimenter/experimenter/nimbus-ui/src/components/PageSummary/TableSignoff/index.tsx

```diff
@@ -1,24 +1,41 @@
-/* This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
-
 import React from "react";
 import { Card, Table } from "react-bootstrap";
+import Form from "react-bootstrap/Form";
 import LinkExternal from "src/components/LinkExternal";
 import { EXTERNAL_URLS } from "src/lib/constants";
 import { getExperiment_experimentBySlug_signoffRecommendations } from "src/types/getExperiment";
 
 type TableSignoffProps = {
   signoffRecommendations: getExperiment_experimentBySlug_signoffRecommendations | null;
+  legalSignoff: boolean | null;
+  qaSignoff: boolean | null;
+  vpSignoff: boolean | null;
+  onLegalSignoffChange: (value: boolean) => void;
+  onQaSignoffChange: (value: boolean) => void;
+  onVpSignoffChange: (value: boolean) => void;
 };
 
-const TableSignoff = ({ signoffRecommendations }: TableSignoffProps) => (
+const TableSignoff = ({
+  signoffRecommendations,
+  legalSignoff,
+  qaSignoff,
+  vpSignoff,
+  onLegalSignoffChange,
+  onQaSignoffChange,
+  onVpSignoffChange,
+}: TableSignoffProps) => (
   <Card.Body>
     <Table data-testid="table-signoff">
       <tbody>
         <tr data-testid="table-signoff-qa">
           <th className="border-top-0">QA Sign-off</th>
-          <td className="border-top-0">
+          <td className="border-top-0 d-flex">
+            <Form.Check
+              type="checkbox"
+              data-testid="is-qasignoff-checkbox"
+              checked={!!qaSignoff}
+              onChange={(e) => onQaSignoffChange(e.target.checked)}
+            />
```

Comment:
While youre here could you fix the text so that there is a space after recommended?

in JSX, `<span>text </span>` the trailing space is stripped, so you need to do `<span>text {" "}</span>`

Reply:
this doesn't seem to be working, i was able to change it `<span className="text-success">Recommended
:&nbsp;</span>` <img width="991" alt="Screenshot 2024-02-29 at 3 40 41 PM" src="https://github.com/mozilla/experimenter/assets/25848231/fb596f3a-bc95-4e0f-afef-0a3f13a7f1b3">

Reply:
Yeah that is super weird, it is supposed to work. The {" "} before Learn more isnt working either so i give up